app/assets/stylesheets/application.css.scss

```diff
@@ -25,21 +29,21 @@
    margin-right: 7px; /*space between*/
  }
 
- .social-links a i {
+.social-links a i {
```

Comment:
Merge rule `.social-links a i` with rule on line 22<br>Selector should have depth of applicability no greater than 2, but was 3